app/components/labs-ui/layer-group-toggle.js

```diff
@@ -39,6 +39,27 @@ export default Component.extend({
   actions: {
     toggle() {
       this.toggleProperty('active');
+      window.dataLayer = window.dataLayer || [];
+      if(this.get('label')==="Thematic Map") {
+        window.dataLayer.push({
+          'event' : 'toggle_thematic_map',
+          'toggle' : this.get('active')
+        });
+      } else if(["Subways", "Zipcodes", "Neighborhood Tabulation Areas", "Community Districts", "Community District Tabulation Areas (CDTAs)", "NYC Council Districts"].includes(this.get('label'))){
+        // Only count toggle on, not toggle off
+        if(this.get('active')) {
+          window.dataLayer.push({
+            'event' : 'toggle_map_layer',
+            'toggle_map_layer' : this.get('label')
+          });
+        }
+
+      } else {
+        window.dataLayer.push({
+          'event' : this.get('label'),
```

Comment:
Why is the event name just the label name here? Should it be something like "toggle_other_layer" ? 
Can you write a comment about what types of layers may be toggled under this else?

Reply:
I am currently unaware of any other instances that would cause this to occur.  I have included a comment to note that this is simply a failsafe.

Reply:
okay, gotcha. Thanks for adding that comment. I think in this case a better event name will be 'other_layer_toggle', otherwise you will get a lot of confusing noise in your event logs. We'll see stuff like 'zoning_layer' for event tags

Reply:
Maybe 
```
window.dataLayer.push({
          'event' : 'other_layer_toggle',
          'layer': this.get('label'),
          'active' : this.get('active')
```

Reply:
I've updated this to create a category to capture any events of this nature.